### ISSUE TITLE
Get more than 20 results for popular, upcoming etc.

### ISSUE THREAD

**Issue by joshkremer** [18Months/themoviedb-api#4]
Great gem, really useful! Can you tell me how I could receive more than 20 results when using popular, upcoming etc? I assume I need to paginate, but I can't find where to do that in your gem. Thanks!

**Comment by samuelebistoletti** [comment]
Thanks. You can pass the param "page" to all methods that supports such paramater, for example you could write
`Tmdb::Movie.popular page: 2`
to get the second page of results.

**Comment by joshkremer** [comment]
Thanks for the reply. I'm unable to make that work. I receive a "wrong number of arguments (1 for 0)" error. Is there something I need to set up first?

**Comment by samuelebistoletti** [comment]
What method are you calling with param page?

**Comment by joshkremer** [comment]
Here is what I'm trying:

toprated = Tmdb::Movie.top_rated page: 2
puts toprated

**Comment by samuelebistoletti** [comment]
I tried the same exact call and it's working for me, no exeception is raised.

Are you using the last available version of the gem? What ruby version are you on?

**Comment by joshkremer** [comment]
Interesting. I'm running ruby 2.2.1p85. I have updated all of my gems and still have the same problem.

**Comment by samuelebistoletti** [comment]
Ok, Ruby version should be ok. Can you please send me the full stack trace of the exception raised?

**Comment by joshkremer** [comment]
Sure, is this what you are looking for?

/Users/joshkremer/.rvm/gems/ruby-2.2.1/gems/themoviedb-1.0.1/lib/themoviedb/movie.rb:71:in `top_rated': wrong number of arguments (1 for 0) (ArgumentError)
	from rebuild.rb:10:in `<main>'

**Comment by samuelebistoletti** [comment]
Yes. As I can see from this output the error seams to be on line 10 of
rebuild.rb, maybe you are calling the method on tmdb gem from that file in
the wrong way. I tried from irb and top_rated method works as usual. Can
you publish a short gist of your code in order to reproduce the problem?
Il 05 apr 2016 14:03, "joshkremer" <notifications@github.com> ha scritto:

**Comment by joshkremer** [comment]
Absolutely, here you go: https://gist.github.com/joshkremer/77ca70119a6086822d10a3f8d8734b11

**Comment by druzn3k** [comment]
I just tryed too and I get the expected results. 

```
 ~/workspace/ruby/scripts $ ruby -v 
> ruby 2.2.1p85 (2015-02-26 revision 49769) [x86_64-linux]
 ~/workspace/ruby/scripts $ ruby tmdb_issue_4_ruby_2.1.1.rb
> #<Tmdb::Movie page=2, results=[#<Tmdb::Movie poster_path="/yRXTVpDRBA3983C3HjoY0SO4dV6.jpg", [...]
```

But I had to correct the `require`, as it wasn't loading the correct gem, `themoviedb-api`, but it was loading `themoviedb`. Those are two different gems, with two different interfaces (but similar class names): [themoviedb](https://rubygems.org/gems/themoviedb) and [themoviedb-api](https://rubygems.org/gems/themoviedb-api).

Hope this solves the mistery! :+1:
